src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IDataBlockService.cs

```diff
@@ -21,26 +19,30 @@ public interface IDataBlockService
 
         Task<Either<ActionResult, Unit>> Delete(ReleaseId releaseId, DataBlockId id);
 
-        Task<Either<ActionResult, DataBlockViewModel>> Get(DataBlockId id);
+        Task<Either<ActionResult, DataBlockViewModel>> Get(DataBlockId dataBlockVersionId);
 
         Task<Either<ActionResult, List<DataBlockSummaryViewModel>>> List(ReleaseId releaseId);
 
         Task<Either<ActionResult, DataBlockViewModel>> Update(DataBlockId id, DataBlockUpdateViewModel dataBlockUpdate);
 
         Task<Either<ActionResult, Unit>> DeleteDataBlocks(DeleteDataBlockPlan deletePlan);
 
-        Task<Either<ActionResult, DeleteDataBlockPlan>> GetDeletePlan(Guid releaseId, Guid dataBlockId);
+        Task<Either<ActionResult, DeleteDataBlockPlan>> GetDeletePlan(ContentSectionId releaseId, ContentSectionId dataBlockId);
```

Comment:
Refactoring in this class doesn't look right to me. For example I would expect this line to be `GetDeletePlan(ReleaseId releaseId, DataBlockId dataBlockId);`, but why are we using this pattern all of a sudden rather than using `Guid`?

Reply:
Good spot - I'd removed the majority of them in previous commits but some imports were still lurking that were aliasing "Guid", and so Rider helpfully stepped in and converted all my Guids back to aliases again!

Made sure that all of the aliases are removed, and converted everything back to good old Guids.